tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Package/SdkReleaseTool.cs

```diff
@@ -63,7 +63,7 @@ public async Task<SdkReleaseResponse> ReleasePackageAsync(string packageName, st
                     response.ReleaseStatusDetails = "Package name cannot be null or empty. ";
                     isValidParams = false;
                 }
-                if (string.IsNullOrWhiteSpace(language) || !ValidLanguages.Contains(language))
+                if (string.IsNullOrWhiteSpace(language) || !ValidLanguages.Contains(language, StringComparer.OrdinalIgnoreCase))
```

Comment:
I haven't checked, but are there any other places where we need to be explicit about it?